config/webpack/react/webpack.config.base.js

```diff
@@ -1,28 +1,28 @@
 'use strict';
```

Comment:
Again, why we prefer webpack over parcel here?

And why not use `.ts` always consistently?